examples/inference_restoring_model_dependencies/README.md

```diff
@@ -0,0 +1,23 @@
+### MLflow evaluation Examples
+
+The examples in this directory illustrate how you can use the `mlflow.pyfunc.spark_udf` API with argument
+`env_manager="conda"` to create a spark UDF for model inference with restoring the python environment used
+in model training.
```

Comment:
@dbczumar @BenWilson2 How would you phrase `model inference with restoring the python environment`)?


- `model inference with dependency restoration`?
- `model inference with environment restoration`?

Reply:
I'd say `to create a spark UDF for model inference that executes in an environment containing the exact dependency versions used during training`